app/forms/applicants/basic_details_form.rb

```diff
@@ -12,17 +11,15 @@ class BasicDetailsForm < BaseForm
     attr_accessor(*ATTRIBUTES)
     attr_writer :date_of_birth
 
-    before_validation :normalise_national_insurance_number
-
     before_validation do
       squish_whitespaces(:first_name, :last_name)
     end
 
     # Note order of validation here determines order they appear on page
     # So validations for each field need to be in order, and presence validations
     # split so that they occur in the right order.
-    validates :first_name, :last_name, presence: true, unless: :draft?
-
+    validates :first_name, presence: true, unless: proc { draft? && last_name.present? }
+    validates :last_name, presence: true, unless: proc { draft? && first_name.present? }
```

Comment:
This was done as it is currently possible to create an application and not provide a first or last name and save as draft. In such situation it is not then possible to select that application from the list of applications